deploy.sh

```diff
@@ -26,7 +30,7 @@ select opt in "${options[@]}"
                 then
                   adb push $opt/usr/share/* /usr/share/
                 else
-                  scp -r $opt/usr/share/* root@192.168.2.15:/usr/share/
+                  echo scp -P${WATCHPORT} -r $opt/usr/share/* root@${WATCHADDR}:/usr/share/
```

Comment:
I think you didn't mean to keep the "echo" here

Reply:
Everything's now fixed up, I believe.